mlir/test/Conversion/GPUToNVVM/gpu-to-nvvm.mlir

```diff
@@ -1058,3 +1058,29 @@ gpu.module @test_module_53 {
     func.return %result32, %result64 : f32, f64
   }
 }
+
+gpu.module @test_module_54 {
+  // CHECK: llvm.func @__nv_isinff(f32) -> i1
```

Comment:
Usually the declarations in libdevice itself return an i32 not an i1. Can we emit i32 to avoid a type error?

Reply:
Presumably we'd emit two ops in this case the call to libdevice then a zero comparison

Reply:
Indeed, thanks!